phpunit.xml.dist

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit colors="true" bootstrap="Tests/bootstrap.php">
+<phpunit colors="true" bootstrap="vendor/autoload.php">
```

Comment:
i think we have a `test-bootstrap.php` file in other bundles. but i am not sure whats the correct thing to do 🙈 
https://github.com/sulu/SuluHeadlessBundle/blob/master/phpunit.xml.dist

Reply:
Yeah you are right, I only checked the theme-bundle and there we used the `autoload.php` 😅